src/Button/test/ButtonSpec.tsx

```diff
@@ -114,4 +114,23 @@ describe('Button', () => {
 
     expect(buttonRef.current).to.have.tagName('button');
   });
+
+  it('Should render `startIcon` before text', () => {
+    render(
+      <Button startIcon={<i>Icon</i>} data-testid="button">
+        Text
+      </Button>
+    );
+
+    expect(screen.getByTestId('button')).to.have.text('IconText');
```

Comment:
The button element can be returned using `screen.getByRole('button')`.

Reply:
OK, tests updated.